wherehows-web/app/controllers/search.ts

```diff
@@ -41,11 +85,13 @@ export default class Search extends Controller.extend({
     } else {
       return 1;
     }
-  }),
-  first: computed('model.page', function() {
-    var model = this.get('model');
+  }
+
+  @computed('model.page')
+  get first(): boolean {
+    const model = this.get('model');
     if (model && model.page) {
-      var currentPage = model.page;
+      const currentPage = model.page;
       if (currentPage <= 1) {
```

Comment:
Couldn't this just be return currentPage <= 1? Why all these weird if blocks taking up space?

Reply:
That... was there too!

Reply:
I am going to leave it for now. We will come back to this later.